modules/nixos/panel.nix

```
@@ -48,6 +55,22 @@ in {
       '';
     };
 
+    panel = mkOption {
+      type = enum ["pterodactyl" "pelican-dev"];
+      default = "pterodactyl";
+      description = ''
+        Panel type to use. Can be either 'pterodactyl' or 'pelican-dev'.
+      '';
```

```suggestion
      description = ''
        Panel type to use.
      '';
```


The type `enum` already indicates possible values. Instead describe what this value will be used for.